### PR TITLE
Radius search fix

### PR DESCRIPTION
**Pull request by kowsar89**

### CODE REVIEW

File: includes/model/SearchForm.php [hunk 1]
```diff
@@ -328,20 +328,20 @@ public function buttons_template() {
 		}
 	}
 
-	public function range_slider_data( $data ) {
```
Comment (obiPlabon) [inline includes/model/SearchForm.php:331]: Please do not delete any method without prior notice. We can deprecate first then after 3/4 releases we can hard deprecate or remove that method.

**Review by obiPlabon — CHANGES_REQUESTED** [review]

File: includes/model/SearchForm.php (hunk 1 above)
Comment (kowsar89) [reply includes/model/SearchForm.php:331]: This method still exists in line 340

Comment (obiPlabon) [reply includes/model/SearchForm.php:331]: Sorry, my bad.

**Review by vairafiq — APPROVED** [review]
Reviewed

**Comment by vairafiq** [comment]
Searched radius is not checked in advanced filter

**Comment by vairafiq** [comment]
Still not working https://www.awesomescreenshot.com/video/9556385?key=cae85b031c32738af28266bd7cb9508b

**Comment by vairafiq** [comment]
Working perfectly now

**Review by obiPlabon — APPROVED** [review]
Working fine

**Pull request merged**